loculus_values/values.yaml

```diff
@@ -2438,3 +2438,25 @@ organisms:
           sequence: "[[URL:https://corneliusroemer.github.io/seqs/artefacts/hmpv/SH.fasta]]"
 ingest:
   mirrorBucket: "https://hel1.your-objectstorage.com/loculus-public/mirror/"
+defaultResources:
+  requests:
+    memory: "10Mi"
+    cpu: "10m"
+  limits:
+    memory: "1Gi"
+    cpu: "1000m"
+
+resources:
+  ingest:
+    requests:
+      memory: "1Gi"
+      cpu: "50m"
```

Comment:
We could also alternatively this in Loculus itself (is `200m`)

Reply:
Ingest is often using the full 100m:
![image](https://github.com/user-attachments/assets/d2ecabe8-8a24-4e9a-89c7-1f631c6cbf08)

Reply:
Feel free to propose changes - I just copied this over from the Loculus values.yaml and reduced the CPU

Reply:
> I would use the same limits from loculus then actually if thats ok

I'm not sure if you are proposing here that we leave out the `limit` - if we did I'm not certain that the limit from the original Loculus values would come through and be combined (this is the problem we had that prompted https://github.com/pathoplexus/pathoplexus/pull/456)